R/plot_temp.R

```diff
@@ -12,13 +17,18 @@
 #'@author
 #'Jordan S. Read, Luke A. Winslow
 #'@examples 
-#'sim_folder <- run_example_sim(verbose = FALSE)
-#'nc_file <- file.path(sim_folder, 'output.nc')
-#'plot_temp(file = nc_file, fig_path = FALSE)
-#'plot_temp(file = nc_file, fig_path = 'test_figure.png', height = 3, reference = 'surface')
+#'nc_file <- system.file("extdata", "output.nc", package = "glmtools")
+#'plot_temp(nc_file = nc_file, fig_path = FALSE)
+#'plot_temp(nc_file = nc_file, fig_path = 'test_figure.png', height = 3, reference = 'surface')
 #'@export
-plot_temp <- function(file='output.nc', fig_path = FALSE, reference = 'surface', col_lim, ...){
-
-  plot_var(file, var_name = 'temp', fig_path, reference, col_lim, ...)
+plot_temp <- function(nc_file='output.nc',fig_path = NULL, reference='surface', legend.title = NULL, 
+                      interval = 0.5, text.size = 12, show.legend = TRUE, 
+                      legend.position = 'right', plot.title = NULL){
+  
+  .Deprecated('plot_var',msg = 'Deprecated. Use `plot_var`, where default is var_name = `temp`')
```

Comment:
Do you get two deprecated warnings here? i think `plot_var` is deprecated too in favor or `plot_var_nc`. Perhaps just plug `plot_var_nc` directly in here instead of pointing to `plot_var` which then points to `plot_var_nc`

Reply:
Yes! good catch